src/main/java/com/hermesworld/ais/galapagos/topics/service/impl/TopicServiceImpl.java

```diff
@@ -62,13 +64,14 @@ public class TopicServiceImpl implements TopicService, InitPerCluster {
 
     public TopicServiceImpl(KafkaClusters kafkaClusters, ApplicationsService applicationsService,
             NamingService namingService, CurrentUserService userService, GalapagosTopicConfig topicSettings,
-            GalapagosEventManager eventManager) {
+            GalapagosEventManager eventManager, MessagesServiceFactory MessagesServiceFactory) {
```

Comment:
Parameter names must start with a lowercase letter! (`messagesServiceFactory` as the variable name)